Frends.ServiceBus.Read/Frends.ServiceBus.Read/Definitions/Result.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+
+namespace Frends.ServiceBus.Read.Definitions;
+
+/// <summary>
+/// Read result.
+/// </summary>
+public class Result
+{
+    /// <summary>
+    /// Read result.
+    /// </summary>
+    public ReadResult Results { get; private set; }
+
+    internal Result(ReadResult results)
+    {
+        Results = results;
+    }
+}
```

Comment:
In my opinion you need to list all the Result objects attributes to help the automatic documentation.

Reply:
This needs to resolved. Following the Frends Tasks guidelines the Result object needs to be documented.